docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+jupyter
```

Comment:
there was an environment.yml there, we should have one or the other, not fussed on which one!

Reply:
oops - OK reverting to environment.yml